labshare/migrations/0022_add_users_to_devices.py

```diff
@@ -0,0 +1,32 @@
+# Generated by Django 2.2.17 on 2020-12-10 09:36
+
+from django.contrib.auth.models import User
+from django.db import migrations
+from rest_framework.authtoken.models import Token
+
+from labshare.models import Device
+
+
+def create_users(apps, schema_editor):
+    devices = Device.objects.all()
```

Comment:
Instead of importing the device you must use sth. like `Person = apps.get_model('yourappname', 'Person')`, in order to use the correct model class at the time of the migration.